src/katello/constants.py

```diff
@@ -13,13 +11,13 @@
 PROFILE_CACHE_FILE = '/var/lib/rhsm/cache/profile.json'
 
 try:
-    imp.find_module('zypp_plugin')
+    import zypp_plugin
     ZYPPER = True
 except ImportError:
     ZYPPER = False
 
 try:
-    imp.find_module('yum')
+    import yum
```

Comment:
If you change it to this, then this part becomes redundant:
https://github.com/Katello/katello-host-tools/blob/ecd9b1a962e2be16243e5d89208ea0dea1f665e6/src/katello/enabled_report.py#L5-L6

Reply:
how so? it would then need to `from katello.constants import yum` instead…

Reply:
Oh yes, Ruby has created some brain damage where I assumed it was present in a global namespace.